includes/themes.php

```diff
@@ -43,6 +43,8 @@ function __construct() {
 			require_once WPBDP_PATH . 'includes/admin/controllers/class-themes-admin.php';
 			$this->admin = new WPBDP_Themes_Admin( $this, wpbdp()->licensing );
 		}
+
+		$this->check_theme_requiered_version();
```

Comment:
```suggestion
		$this->check_theme_required_version();
```
Just a typo here too.